CHANGELOG.md

```diff
@@ -0,0 +1,22 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
+
+## [1.1.0]
```

Comment:
@patrickjahns 1.1.0?

Reply:
thanks - didn't spot this before - seems to be a typo by @PVince81 
Will fix

Reply:
🤔 - would a bad changelog trigger a 1.0.1 release?

Reply:
probably not worth it... but can keep it in the pipe of change for whatever the next release will be...